sconfig/shared/src/main/scala/org/ekrich/config/impl/AbstractConfigValue.scala

```diff
@@ -350,10 +350,20 @@ abstract class AbstractConfigValue private[impl] (val _origin: ConfigOrigin)
       indent: Int,
       atRoot: Boolean,
       options: ConfigRenderOptions
-  ): Unit = {
-    val u = unwrapped
-    sb.append(u.toString)
-  }
+  ): Unit =
+    if (hideEnvVariableValue(options)) {
+      sb.append("<env variable>")
+    } else {
+      val u = unwrapped
+      sb.append(u.toString)
+    }
+
+  protected def hideEnvVariableValue(options: ConfigRenderOptions): Boolean =
+    !options.getShowEnvVariableValues && (origin.originType eq OriginType.ENV_VARIABLE)
```

Comment:
```suggestion
    !options.getShowEnvVariableValues() && (origin.originType eq OriginType.ENV_VARIABLE)
```
To fix that in CI

Reply:
I've changed `getShowEnvVariableValues` method. Now it's like other gets in the same file.

Reply:
sure, I know it. Higher precedence has consistency in the repo.